src/Form/index.ts

```diff
@@ -41,7 +41,7 @@ export {
 
 export type TypeFieldState = 'default' | 'disabled' | 'required' | 'valid' | 'invalid' | 'processing';
 export type TypeButtonDesigns = 'primary' | 'secondary' | 'danger';
-export type TypeButtonSizes = 'small' | 'normal' | 'large';
+export type TypeButtonSizes = 'xSmall' | 'small' | 'normal' | 'large';
```

Comment:
Can keep this all lower case? I think `xsmall` is more consistent and matches the `xlarge` that `SpinnerSize` uses.